### ISSUE TITLE
bedtools jaccard complains because chromosome is not represented in both query and database files 

### ISSUE THREAD

**Issue by nw11** [arq5x/bedtools2#616]
Using bedtools jaccard gives me the error below when one of the files contains a sequence id (chrM in this case) that the other does not. 

E.g.
```
ERROR:    pk.bed.gz contains chromosome chrM, but the query file does not.
       Please re-reun with the -g option for a genome file.
       See documentation for details.
```
Why should it matter whether the same chromosomes are represented in the two files or not?
bedtools 2.25

**Comment by scalavision** [comment]
I also get this error: bedtools 2.27.1

Just skipping the chromosomes not present in both files could be a solution right? Eventually make a log of what chromosomes where skipped ...

**Comment by arq5x** [comment]
I cannot reproduce this. Can you send an example?

**Comment by karl616** [comment]
I have the error as well. bedtools v2.28.0, installed through bioconda. I don't even get the error... just a segmentation error thrown my way.

Here are two example files that doesn't work for me:

[sampleA.txt](https://github.com/arq5x/bedtools2/files/3083810/sampleA.txt)
[sampleB.txt](https://github.com/arq5x/bedtools2/files/3083811/sampleB.txt)

**Comment by arq5x** [comment]
I cannot reproduce this. I suspect it is an outdated version. I will update the @bioconda version after the next release

**Comment by rikrdo89** [comment]
Was this ever fixed? I tried the latest version available though conda (bedtools v2.31.1), and I still see this error when a chromosome is not present in one of the files.

```
ERROR: Database file test.bed contains chromosome chrY, but the query file does not.
       Please rerun with the -g option for a genome file.  
       See documentation for details.
```

**Comment by arq5x** [comment]
Hmm, do you have example files I could test with?
